include/swift/AST/DiagnosticsSema.def

```diff
@@ -2089,7 +2089,7 @@ ERROR(property_behavior_protocol_no_initStorage,none,
       (Type, Type))
 ERROR(property_behavior_unknown_requirement,none,
       "property behavior protocol %0 has non-behavior requirement %1",
-      (Identifier, Identifier))
+      (Identifier, DeclBaseName))
```

Comment:
@jckarter I changed this property behaviour related diagnostic I changed to `DeclBaseName`. Is that right or should it stay an `Identifier`?